packages/db-migrations/databases/fxa/patches/patch-142-143.sql

```diff
@@ -0,0 +1,87 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('142');
+
+-- Adds the providerId which maps to the third party auth provider used to generate session
+ALTER TABLE `sessionTokens`
+ADD COLUMN `providerId` TINYINT DEFAULT NULL,
```

Comment:
I guessing the idea is that we are going to use the values in this column to find sessions to revoke so I think we should index this column.

Reply:
@chenba I thought about adding an index here but opted not too since the lookup would be on `uid` (which is index) and `provider`. This the table is pretty big, didn't want to risk any performance impact on adding a new index.